2025-03-09 18:36:35,953 INFO    MainThread:21344 [wandb_init.py:setup_run_log_directory():647] Logging user logs to c:\Users\prajv\Downloads\ass1\DA6401_ASSIGNMENT1\wandb\run-20250309_183635-ayubj2te\logs\debug.log
2025-03-09 18:36:35,953 INFO    MainThread:21344 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to c:\Users\prajv\Downloads\ass1\DA6401_ASSIGNMENT1\wandb\run-20250309_183635-ayubj2te\logs\debug-internal.log
2025-03-09 18:36:35,953 INFO    MainThread:21344 [wandb_init.py:monkeypatch_ipython():599] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001CA7AF9C260>
2025-03-09 18:36:35,953 INFO    MainThread:21344 [wandb_init.py:init():761] calling init triggers
2025-03-09 18:36:35,953 INFO    MainThread:21344 [wandb_init.py:init():766] wandb.init called with sweep_config: {}
config: {'_wandb': {}}
2025-03-09 18:36:35,953 INFO    MainThread:21344 [wandb_init.py:init():784] starting backend
2025-03-09 18:36:35,953 INFO    MainThread:21344 [wandb_init.py:init():788] sending inform_init request
2025-03-09 18:36:35,964 INFO    MainThread:21344 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=spawn, using: spawn
2025-03-09 18:36:35,964 INFO    MainThread:21344 [wandb_init.py:init():798] backend started and connected
2025-03-09 18:36:35,969 INFO    MainThread:21344 [wandb_run.py:_label_probe_notebook():1204] probe notebook
2025-03-09 18:36:35,995 INFO    MainThread:21344 [wandb_run.py:_label_probe_notebook():1214] Unable to probe notebook: 'charmap' codec can't decode byte 0x81 in position 8040: character maps to <undefined>
2025-03-09 18:36:35,995 INFO    MainThread:21344 [wandb_init.py:init():891] updated telemetry
2025-03-09 18:36:36,065 INFO    MainThread:21344 [wandb_init.py:init():915] communicating run to backend with 90.0 second timeout
2025-03-09 18:36:37,732 INFO    MainThread:21344 [wandb_init.py:init():990] starting run threads in backend
2025-03-09 18:36:38,131 INFO    MainThread:21344 [wandb_run.py:_console_start():2375] atexit reg
2025-03-09 18:36:38,133 INFO    MainThread:21344 [wandb_run.py:_redirect():2227] redirect: wrap_raw
2025-03-09 18:36:38,133 INFO    MainThread:21344 [wandb_run.py:_redirect():2292] Wrapping output streams.
2025-03-09 18:36:38,133 INFO    MainThread:21344 [wandb_run.py:_redirect():2315] Redirects installed.
2025-03-09 18:36:38,135 INFO    MainThread:21344 [wandb_init.py:init():1032] run started, returning control to user process
2025-03-09 18:36:38,135 INFO    MainThread:21344 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-09 18:36:38,135 INFO    MainThread:21344 [wandb_init.py:_pause_backend():564] pausing backend
2025-03-09 18:36:38,146 INFO    MainThread:21344 [wandb_init.py:_resume_backend():569] resuming backend
2025-03-09 18:36:38,153 INFO    MainThread:21344 [wandb_config.py:__setitem__():154] config set learning_rate = 0.01 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000001CA7AF9D160>>
2025-03-09 18:36:38,153 INFO    MainThread:21344 [wandb_run.py:_config_callback():1261] config_cb learning_rate 0.01 None
2025-03-09 18:36:38,154 INFO    MainThread:21344 [wandb_config.py:__setitem__():154] config set epochs = 10 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000001CA7AF9D160>>
2025-03-09 18:36:38,154 INFO    MainThread:21344 [wandb_run.py:_config_callback():1261] config_cb epochs 10 None
2025-03-09 18:36:38,154 INFO    MainThread:21344 [wandb_config.py:__setitem__():154] config set batch_size = 64 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000001CA7AF9D160>>
2025-03-09 18:36:38,154 INFO    MainThread:21344 [wandb_run.py:_config_callback():1261] config_cb batch_size 64 None
2025-03-09 18:36:38,155 INFO    MainThread:21344 [wandb_config.py:__setitem__():154] config set hidden_layers = [256, 128, 64] - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000001CA7AF9D160>>
2025-03-09 18:36:38,155 INFO    MainThread:21344 [wandb_run.py:_config_callback():1261] config_cb hidden_layers [256, 128, 64] None
2025-03-09 18:36:38,156 INFO    MainThread:21344 [wandb_config.py:__setitem__():154] config set input_size = 784 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000001CA7AF9D160>>
2025-03-09 18:36:38,156 INFO    MainThread:21344 [wandb_run.py:_config_callback():1261] config_cb input_size 784 None
2025-03-09 18:36:38,156 INFO    MainThread:21344 [wandb_config.py:__setitem__():154] config set output_size = 10 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000001CA7AF9D160>>
2025-03-09 18:36:38,156 INFO    MainThread:21344 [wandb_run.py:_config_callback():1261] config_cb output_size 10 None
2025-03-09 18:36:38,157 INFO    MainThread:21344 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-09 18:36:38,157 INFO    MainThread:21344 [wandb_init.py:_pause_backend():564] pausing backend
2025-03-09 18:36:41,485 INFO    MainThread:21344 [wandb_init.py:_resume_backend():569] resuming backend
2025-03-09 18:36:42,233 INFO    MainThread:21344 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-09 18:36:42,233 INFO    MainThread:21344 [wandb_init.py:_pause_backend():564] pausing backend
2025-03-09 18:36:42,271 INFO    MainThread:21344 [wandb_init.py:_resume_backend():569] resuming backend
2025-03-09 18:37:52,737 INFO    MainThread:21344 [wandb_run.py:_finish():2112] finishing run da6401-assignments/assignment1/ayubj2te
2025-03-09 18:37:52,740 ERROR   MainThread:21344 [jupyter.py:save_history():450] Run pip install nbformat to save notebook history
2025-03-09 18:37:52,740 INFO    MainThread:21344 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-09 18:37:52,740 INFO    MainThread:21344 [wandb_init.py:_jupyter_teardown():581] cleaning up jupyter logic
2025-03-09 18:37:52,740 INFO    MainThread:21344 [wandb_run.py:_atexit_cleanup():2340] got exitcode: 0
2025-03-09 18:37:52,741 INFO    MainThread:21344 [wandb_run.py:_restore():2322] restore
2025-03-09 18:37:52,741 INFO    MainThread:21344 [wandb_run.py:_restore():2328] restore done
2025-03-09 18:37:54,652 INFO    MainThread:21344 [wandb_run.py:_footer_history_summary_info():3956] rendering history
2025-03-09 18:37:54,652 INFO    MainThread:21344 [wandb_run.py:_footer_history_summary_info():3988] rendering summary
2025-03-09 18:37:54,656 INFO    MainThread:21344 [wandb_run.py:_footer_sync_info():3917] logging synced files
